src/Core/Grid/Presenter/GridPresenter.php

```diff
@@ -69,6 +79,13 @@ public function present(GridInterface $grid)
             ],
             'filters' => $searchCriteria->getFilters(),
         ];
+
+        $this->hookDispatcher->dispatchForParameters('action'.$definition->getId().'GridPresenterModifier', [
+            'presentedGrid' => &$presentedGrid,
+            'grid' => $grid,
+        ]);
+
+        return $presentedGrid;
```

Comment:
according to our schema, it should produce immutable view, but array is mutable. What if we had `GridView` instance? Similar as Forms do, except much simpler.

Reply:
later :)